specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json

```diff
@@ -367,6 +367,10 @@
           "items": {
             "$ref": "#/definitions/MetadataDTO"
           }
+        },
+        "strictFiltersCompoundOperationType": {
+          "type": "integer",
```

Comment:
why integer?, its enum right?

Reply:
Modified to enum

---
In reply to: [454802745](https://github.com/Azure/azure-rest-api-specs/pull/10132#discussion_r454802745) [](ancestors = 454802745)